### PR TITLE
first commit / catalog hovers

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
- [DEMO LINK](https://NikitaKarbazin.github.io/layout_catalog_hovers/)

### CODE REVIEW

**Comment by Kaunin1** [comment]
Good job  !

**Review by Kaunin1 — APPROVED** [review]

**Review by illiarerun — CHANGES_REQUESTED** [review]
GJ ,little fixes left:

File: src/styles/_header.scss [hunk 1]
```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```
[the remark this answers is not in the document]
Comment (NikitaKarbazin) [reply src/styles/_header.scss]: fixed

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```
[the remark this answers is not in the document]
Comment (NikitaKarbazin) [reply src/styles/_catalog.scss]: ok, i understand my mistake)

**Review by mkyloa — APPROVED** [review]

**Pull request closed without merging**